addons/expanded-backpack/addon.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Multiple rows in backpack",
+  "description": "Display the contents of the editor backpack in an expandable two-way grid similar to the costume library and increase the size of thumbnails to improve ease of use.",
+  "credits": [
+    {
+      "name": "pumpkinhasapatch",
+      "link": "https://github.com/pumpkinhasapatch"
+    }
+  ],
+  "versionAdded": "1.29.0",
+  "tags": ["editor"],
+  "enabledByDefault": false,
+  "settings": [
+    {
+      "name": "Number of displayed rows",
+      "id": "rows",
+      "type": "integer",
+      "min": 1,
+      "max": 4,
+      "default": 2
+    },
+    {
+      "name": "Improved costume and script previews",
+      "id": "upscale",
+      "type": "boolean",
+      "default": true
+    }
+  ],
+  "customCssVariables": [
+    {
+      "name": "imgsize",
+      "value": {
+        "type": "ternary",
+        "source": {
+          "type": "settingValue",
+          "settingId": "upscale"
+        },
+        "true": "95%",
+        "false": "32px"
+      }
+    },
+    {
+      "name": "bgcolor",
+      "value": {
+        "type": "ternary",
+        "source": {
+          "type": "settingValue",
+          "settingId": "upscale"
+        },
+        "true": "white",
+        "false": "none"
+      }
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "backpack.css",
+      "matches": ["projects"]
+    }
+  ],
+  "updateUserstylesOnSettingsChange": true,
```

Comment:
I don't think `updateUserstylesOnSettingsChange` is necessary for this addon. This property only affects userstyles that use the "if" property inside userstyle definitions. Custom CSS variables are always updated when settings change, and that can't be turned on nor off.